index.html

```diff
@@ -22740,10 +22740,10 @@ <h2 id="terminology">
   <table class="termlist">
       <thead>
         <tr>
-          <th><span its-locale-filter-list="en" lang="en">Terminology</span><br/><span its-locale-filter-list="ja" lang="ja">用語（英語）</span></th>
-          <th><span its-locale-filter-list="en" lang="en">Japanese</span><br/><span its-locale-filter-list="ja" lang="ja">用語（日本語）</span></th>
-          <th><span its-locale-filter-list="en" lang="en">Transliteration</span><br/><span its-locale-filter-list="ja" lang="ja">よみ</span></th>
-          <th><span its-locale-filter-list="en" lang="en">Definition</span><br/><span its-locale-filter-list="ja" lang="ja">定義</span></th>
+          <th><span its-locale-filter-list="en" lang="en">Terminology</span> <span its-locale-filter-list="ja" lang="ja">用語（英語）</span></th>
+          <th class="colminwidth"><span its-locale-filter-list="en" lang="en">Japanese</span> <span its-locale-filter-list="ja" lang="ja">用語（日本語）</span></th>
```

Comment:
We should also remove the class from HTML.

Reply:
Thank you for pointing. Fixed in 7e26372.